### ISSUE TITLE
Is it possible to update TileLayer opacity?

### ISSUE THREAD

**Issue by petervojtek** [PaulLeCam/react-leaflet#304]
### Expected behavior

when I update `opacity` in props, I expect the TileLayer to automatically change its opacity based on the value.

### Actual behavior

The opacity seems to be set when TileLayer is initialized, but then it is not updated when props.opacity is altered. I guess this [react-leaflet code](https://github.com/PaulLeCam/react-leaflet/blob/master/src/TileLayer.js#L22) would have to look like this:

 ```
 updateLeafletElement (fromProps: Object, toProps: Object) {
    if (toProps.url !== fromProps.url) {
      this.leafletElement.setUrl(toProps.url)
    }

   // I guess this is missing in current code to support the opacity update
   if(toProps.opacity !== fromProps.opacity) {
       this.leafletElement.setOpacity(toProps.opacity)
   }
}
```

Maybe there is some good reason why it is not there. I see that in the past this was already an issue in #151.

**Comment by petervojtek** [comment]
hmm, now I see the setOpacity is [called in super-class](https://github.com/PaulLeCam/react-leaflet/blob/master/src/GridLayer.js#L32). I Will have to double check my code, as it is more likely that is where the issue can be.

**Comment by petervojtek** [comment]
ok, so it seems the root of issue may be the same as was in #151, i.e. that `TileLayer .updateLeafletElement ` is missing something like this: `super.updateLeafletElement(fromProps, toProps);` and thus the `GridLayer.setOpacity` is not executed.

**Comment by PaulLeCam** [comment]
Thanks for reporting this, should be fixed in v1.1.4.

**Comment by petervojtek** [comment]
thanks, I confirm 1.1.4 fixed this issue
